cmd/traffic_cache_tool/CacheTool.cc

```diff
@@ -349,8 +346,10 @@ Stripe::updateHeaderFooter()
     for (int j = 0; j < 2; j++) {
```

Comment:
Here, it would be clearer if you did something like
```
  for (auto i : {A, B}) {
    for (auto j : {HEAD, FOOT}) {
```

Reply:
okay